torchbenchmark/util/model.py

```diff
@@ -48,6 +50,18 @@ def _set_mode(self, train):
         (model, _) = self.get_module()
         model.train(train)
 
+    @staticmethod
```

Comment:
I've been considering whether we should provide a standard set of labels such as these that models can use and we can pick assume to have a certain meaning in the profiler / report - what do you think about such an idea?